helpdesk/views/staff.py

```diff
@@ -1695,6 +1701,42 @@ def ticket_dependency_del(request, ticket_id, dependency_id):
 ticket_dependency_del = staff_member_required(ticket_dependency_del)
 
 
+@helpdesk_staff_member_required
+def ticket_resolves_add(request, ticket_id):
+    depends_on = get_object_or_404(Ticket, id=ticket_id)
+    ticket_perm_check(request, depends_on)
+    if request.method == 'POST':
+        form = TicketResolvesForm(depends_on, request.POST)
+        if form.is_valid():
+            ticketdependency = form.save(commit=False)
+            ticketdependency.depends_on = depends_on
+            if ticketdependency.ticket != ticketdependency.depends_on:
+                ticketdependency.save()
+            return HttpResponseRedirect(reverse('helpdesk:view', args=[depends_on.id]))
```

Comment:
You use shortcut like this I believe:

```suggestion
            return redirect(depends_on.get_absolute_url())
```

Reply:
Not my code. Shall I change it anyway?

Reply:
As you want, maybe this could be done for each occurrence in another MR. It's been a while since I'm thinking about refactoring/modernizing the code base (especially views and better formatting templates).

Reply:
Changed in two places, checked in, test are running, the first test already passed.